migrations/sql/20180103142001_initial_schema.sql

```diff
@@ -64,14 +64,14 @@ CREATE TABLE IF NOT EXISTS user_edge (
 );
 
 CREATE TABLE IF NOT EXISTS notification (
-    PRIMARY KEY (id),
-    id              UUID            NOT NULL,
+    PRIMARY KEY (user_id, create_time ASC, id),
```

Comment:
I believe `ASC` is the default order. Shall we omit it here?

Reply:
@zyro `ASC` is left in place for explicitness because the natural order should be `DESC` for notifications but the query analyser with cockroachdb produces really poor query plans when not `ASC` ordered with keyset `WHERE` clauses. It might be useful to leave a note next to it I guess.